src/libasr/codegen/asr_to_cpp.cpp

```diff
@@ -640,6 +645,10 @@ Kokkos::View<T*> from_std_vector(const std::vector<T> &v)
                 src = "std::real(" + src + ")";
                 break;
             }
+            case (ASR::cast_kindType::IntegerToInteger) : {
+                // src = src;
```

Comment:
Why `IntegerToInteger` cast is skipped?

Reply:
cpp internally handles `int` <-> `long long` conversions